### DISCUSSION TITLE
Hey! I want to know if there are any updates on xiangqi

### DISCUSSION THREAD

**Discussion by jeffnier1989** [fairy-stockfish/Fairy-Stockfish#702] [General]
I noticed that the previous nnue file has not been updated for a year

**Comment by ianfab** [comment]
The NNUEs are trained in a decentralized way by the community, so there isn't any schedule on when which variant will be trained next, it happens whenever someone picks it up. However, for xiangqi specifically the training has gone already quite far, so it also gets increasingly harder to improve. 

Furthermore, now that with [Pikafish](https://github.com/official-pikafish/Pikafish) there is a strong specialized open source xiangqi engine based on SF and inspired by Fairy-SF there is also less focus on xiangqi, since that use case is more or less covered by a different open source project already.
